hooks/boot/02-message-helpers.rb

```diff
@@ -6,7 +6,7 @@ def success_message
 
     def server_success_message(kafo)
       success_message
-      say "  * <%= color('Katello', :info) %> is running at <%= color('#{kafo.param('foreman', 'foreman_url').value}', :info) %>"
+      say "  * <%= color('Foreman', :info) %> is running at <%= color('#{kafo.param('foreman','foreman_url').value}', :info) %>"
```

Comment:
Rubocop will should at you for removing the space :)
```suggestion
      say "  * <%= color('Foreman', :info) %> is running at <%= color('#{kafo.param('foreman', 'foreman_url').value}', :info) %>"
```